src/expr/core/src/aggregate/user_defined.rs

```diff
@@ -42,8 +42,11 @@ impl AggregateFunction for UserDefinedAggregateFunction {
 
     /// Creates an initial state of the aggregate function.
     fn create_state(&self) -> Result<AggregateState> {
-        let state = self.runtime.call_agg_create_state()?;
-        Ok(AggregateState::Any(Box::new(State(state))))
+        // FIXME(eric): This is bad. Let's make `create_state` async later
+        futures::executor::block_on(async {
+            let state = self.runtime.call_agg_create_state().await?;
+            Ok(AggregateState::Any(Box::new(State(state))))
+        })
```

Comment:
Since we don't support `async function create_state` for now, what about using `now_or_never` instead of `block_on` here?